### PR TITLE
Travis-CI integration!

### PR DESCRIPTION
**Pull request by YashdalfTheGray**
We're going to have to add the little travis CI status indicator once this is on master. 

This will run tests every time anything is committed to the repo through commits and through PRs, this way we can make sure code coming in is actually good!

Requested reviewers - @Mctalian @davethenipper

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "0.10"
```
Comment (Mctalian) [inline .travis.yml]: I would recommend adding a subminor here as well. Easier to reproduce builds. Similar thought to our bower and package.json. Keeps any changes in libraries from breaking our app as well.

**Comment by Mctalian** [comment]
:+1 Approved.

**Comment by davethenipper** [comment]
Approved

**Comment by YashdalfTheGray** [comment]
Added a patch version to node, merging now.

**Pull request merged**